src/Evolution/Systems/ScalarAdvection/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+
+#include "Domain/BoundaryConditions/BoundaryCondition.hpp"
+#include "Domain/BoundaryConditions/Periodic.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \brief Boundary conditions for the ScalarAdvection system
```

Comment:
You probably meant to have this documentation on the `BoundaryConditions` namespace. Could you move it down a line please?

Reply:
Sure. I also added missing `<cstddef>` :)